metricq/rpc.py

```diff
@@ -90,8 +90,14 @@ async def rpc_dispatch(self, function, **kwargs):
 
 
 def rpc_handler(*function_tags):
-    """
-    Decorator for an RPC handler, may contain multiple functions
+    """A Decorator to mark a method as an RPC handler
+
+    Arguments:
+        *function_tags:
+            The names of the RPCs that this method should handle
+
+    Note:
+        This only has an effect on methods of classes subclassing :class:`RPCDispatcher`.
```

Comment:
While it is technically correct to mention `RPCDispatcher` it's not helpful as neither the class is included in the documentation, nor is it practical to look into every MetricQ class to find the complex inheritance structure. I'd suggest listing the prominent classes (Sink, Source, IntervalSource, HistoryClient,... ) as well.